### PR TITLE
Merge main into dev-1.29

### PR DESCRIPTION
**Pull request by sftim**
Account for the latest updates, including https://github.com/kubernetes/website/pull/43346.

### CODE REVIEW

**Comment by sftim** [comment]
:warning: Please check I didn't miss anything before approving this :warning:

It was a bit rushed.

File: content/en/docs/concepts/workloads/pods/init-containers.md [hunk 1]
```diff
@@ -289,51 +311,9 @@ The Pod which is already running correctly would be killed by `activeDeadlineSec
...
-
-{{% code_sample language="yaml" file="application/deployment-sidecar.yaml" %}}
-
-This feature is also useful for running Jobs with sidecars, as the sidecar
-container will not prevent the Job from completing after the main container
-has finished.
-
-Here's an example of a Job with two containers, one of which is a sidecar:
-
-{{% code_sample language="yaml" file="application/job/job-sidecar.yaml" %}}
```
Comment (sftim) [inline content/en/docs/concepts/workloads/pods/init-containers.md:332]: Moved per https://github.com/kubernetes/website/pull/43346

**Comment by sftim** [comment]
⚠️ Please check I didn't miss anything before approving this ⚠️

It was a bit rushed.

**Comment by katcosgrove** [comment]
Looks clean, other than the nit we can address later.

/lgtm

**Review by natalisucks — APPROVED** [review]
looks good to me, too
/approve

**Pull request merged**